src/pages/index.page.jsx

```diff
@@ -1,7 +1,7 @@
-import Head from "next/head";
-import Banner from "../components/Banner";
-import Image from "next/image";
-import logo from "../../public/reactdevske.svg";
+import Head from 'next/head';
+// import Banner from "../components/Banner";   // commented out due to lint error during lint configuration
```

Comment:
We can remove the comment as well since the component is not used in this file.

Reply:
done ✅